strawberry_django/mutations/resolvers.py

```diff
@@ -655,11 +686,7 @@ def update_m2m(
                 data.pop(key_attr, None)
                 to_add.append(obj)
             elif data:
-                if key_attr not in data:
-                    manager.get_or_create(**data)
-                else:
-                    data.pop(key_attr)
-                    manager.create(**data)
+                create_nested(data)
```

Comment:
I don't fully understand the previous implementation introduced in #449 , why do we need to check input type and use `get_or_create` or just `create`? I tried to unify it regardless the input type, anyway, could you please pay attention to this part of code before changes?